src/CollAction/Models/VideoLink.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace CollAction.Models
+{
+    public class VideoLink
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+
+        [Required]
+        public string Link { get; set; }
```

Comment:
MaxLength 2083? See: http://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Some sort of maximum string length is a bit more efficient in a table